clients/drcachesim/tests/trace_invariants.h

```diff
@@ -36,8 +36,8 @@
 #ifndef _TRACE_INVARIANTS_H_
 #define _TRACE_INVARIANTS_H_ 1
 
-#include "../analysis_tool.h"
```

Comment:
Did this file move? And that's why changing the paths to the included files works? The Github review UI doesn't show a move here though (don't remember how that looks like).

Reply:
Nothing moved: the build has include dir flags that point to all the directories with headers, so the compiler finds this without the explicit path (which makes it hard to embed or use in other build systems).